### PR TITLE
Uds 1411 - Cards need to have links in titles

### PR DESCRIPTION
**Pull request by davidornelas11**
### Description
Cards need to have titles that are links

Titles on cards for events and news now have links in the titles

### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-1411?atlOrigin=eyJpIjoiY2QyNjA5ZmVlMzNmNGVjNTg0Yzc5ZDhhMThlYjllY2YiLCJwIjoiaiJ9))

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples
- [x] No new console errors
- [x] Accessibility checks

### CODE REVIEW

**Review by mlsamuelson — COMMENTED** [review]
For `component-events` we need to be using the `alias` field for the event title's link. We can't count on the rsvp links to be there.

File: packages/component-carousel/src/components/CardCarousel/index.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const htmlTemplate = (
       eventTime={eventTime}
       buttons={buttons}
       linkLabel={linkLabel}
-      linkUrl={linkUrl}
+      linkUrl={linkUrl || buttons?.[0]?.href}
```
Comment (mlsamuelson) [inline packages/component-carousel/src/components/CardCarousel/index.js:60]: @davidornelas11 I debated on whether this update belonged in here, but after testing, seeing that it brings the behavior in-line for the header link regarding the "read more" `linkUrl` and the `buttons`, I think ultimately it makes sense. Good call.

File: packages/components-core/src/components/Card/index.js [hunk 2]
```diff
@@ -316,9 +316,10 @@ const CardContent = ({
   tags,
 }) => (
   <>
+  {console.log(linkUrl, linkLabel)}
```
Comment (mlsamuelson) [inline packages/components-core/src/components/Card/index.js]: `console.log()`

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
@davidornelas11 Still needs work. We should be using using the `alias` field for the event title's link in `component-events`. We can't count on the rsvp links to be there - instead we should be linking to the `alias` which is the url for the event, so it'll always be there.

File: packages/component-events/src/core/transformers/data.transfromer.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const transformData = data => ({
   interests: data.node.interests,
   startDate: data.node.very_start_date,
   endDate: data.node.very_end_date,
-  alias: data.node.alias,
+  alias: data.node.alias, // Also used for card link in title of card
```
Comment (davidornelas11) [inline packages/component-events/src/core/transformers/data.transfromer.js:15]: Kept this the same, only added a comment for clarity on how we use it

File: packages/component-news/src/components/CardGridNews/index.js [hunk 4]
```diff
@@ -24,13 +24,14 @@ const gridRow = (feed, cardButton) => (
       eventFormat="inline"
       eventLocation={feed.location}
       clickable={!!feed.buttonLink}
+      cardLink={feed.alias}
```
Comment (davidornelas11) [inline packages/component-news/src/components/CardGridNews/index.js:27]: matches what we do in component-events, but uses alias without the date, which does not seem to be available for news data

**Review by scott-williams-az — APPROVED** [review]

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**